src/IpHelper.php

```diff
@@ -45,6 +45,30 @@ final class IpHelper
      * The length of IPv4 address in bits.
      */
     public const IPV4_ADDRESS_LENGTH = 32;
+    /**
+     * IP address pattern (for both IPv4 and IPv6 versions). This pattern is PHP and Javascript compatible.
+     * Allows to define your own IP regexp.
+     */
+    public const IP_PATTERN = '((' . self::IPV4_PATTERN . ')|(' . self::IPV6_PATTERN . '))';
+    /**
+     * IP address regexp (for both IPv4 and IPv6 versions). This regexp is PHP and JavaScript compatible.
+     */
+    public const IP_REGEXP = '/^' . self::IP_PATTERN . '$/';
+
+    public static function isIpv4(string $value): bool
+    {
+        return preg_match(self::IPV4_REGEXP, $value) !== 0;
```

Comment:
```suggestion
        return preg_match(self::IPV4_REGEXP, $value) === 1;
```

Result may be `false`: https://www.php.net/manual/ru/function.preg-match.php#refsect1-function.preg-match-returnvalues